torchaudio/_extension.py

```diff
@@ -14,15 +14,34 @@ def _get_lib_path(lib: str):
     return path
 
 
-def _load_lib(lib: str):
+def _load_lib(lib: str) -> bool:
+    """Load extension module
+
+    Note:
+        In case `torchaudio` is deployed with `pex` format, the library file does not
+        exist as a stand alone file.
+        In this case, we expect that `libtorchaudio` is available somewhere
+        in the search path of dynamic loading mechanism, so that importing
+        `_torchaudio` will have library loader find and load `libtorchaudio`.
+        This is the reason why the function should not raising an error when the library
+        file is not found.
+
+    Returns:
+        bool:
+            False if the library file is not found.
+            True if the library file is found AND the library loaded without failure.
```

Comment:
To account for the scenario in which the library file is found but the library fails to load correctly:

```suggestion            
            True if the library file is found AND the library loaded without failure.
            False otherwise.
```

Reply:
Hmm, the suggested change might give an impression that the function still returns when the library is found but not loaded. I would like to make it explicit specification that this function has three branches, 1. the target library is success loaded, 2. the target library was not found (a condition happens in anticipated use case) 3. an issue occurred when loading the library (an unexpected condition, thus we won't try to recover to the normal execution path.)

Reply:
got it, so the raises Exception case covers case 3 right, when the library file is found but there's a failure when loading. can you mention that in the raises exception section just to make it a little bit more clear?

Reply:
How does it sound now?